metricbeat/module/munin/_meta/fields.yml

```diff
@@ -1,7 +1,7 @@
 - key: munin
   title: "Munin"
   description: >
-    experimental[]
+    beta[]
```

Comment:
What does this line do? Asking because I don't see it in the `traefik` module's `fields.yml` file.

Reply:
It adds the header you can see here: https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-munin-node.html But it is now a duplicate as we automatically add it based on the fields.yml. Will clean it up.